lib/module.js

```
@@ -225,8 +226,32 @@ class ModuleCache {
 		return modules;
 	}
 
+	async _checkForDraftRelease(version) {
+		let draftReleaseFound = false;
+		console.log("Check for draft releases by hitting the (private) draft release API endpoint which requires GITHUB_TOKEN");
```

The logger instance that is passed to the `ModuleCache`'s constructor should be used instead of `console.log()`. Same [here](https://github.com/particle-iot/device-os-flash-util/pull/4/files#diff-cdd77182e73349d6c83fbbce1cccf6f15972fbece4dbd9c52eececf1ca3bfcd5R241) and [here](https://github.com/particle-iot/device-os-flash-util/pull/4/files#diff-cdd77182e73349d6c83fbbce1cccf6f15972fbece4dbd9c52eececf1ca3bfcd5R244).

Also, this message is unnecessarily verbose. It would be better to fail early if a draft release is requested but the API access token is not specified (see my other comment about the `--draft` argument).